operations/kubernetes/encryption/README.md

```diff
@@ -31,6 +31,10 @@ For further information, refer to the [official Kubernetes documentation](https:
 
 ## Enabling encryption
 
+When enabling encryption it is important to ensure all 3 nodes are enabled in short order. However that does not mean all control plane nodes should run the script in parallel.
```

Comment:
Some where in here, could add a paragraph that says:
Before encryption is enabled, it is recommend that a Bare-Metal etcd backup be saved (etcd backups are deleted after 24 hours) and that the health of the ectd cluster be verified.
See [Create_a_Manual_Backup_of_a_Healthy_Bare-Metal_etcd_Cluster.md](https://github.com/Cray-HPE/docs-csm/blob/70dc14a21024f4dd20f9416fdd5010c17a0b019b/operations/kubernetes/Create_a_Manual_Backup_of_a_Healthy_Bare-Metal_etcd_Cluster.md)
When encryption is enabled a Bare-Metal etcd cluster can not be restored from a backup taken before encryption is enabled. Such a backup can be used to restore etcd in the event that encryption is later disabled.

Reply:
Pushed up a commit with most of this in place. Let me know if you think that addresses things.